syntax="proto3";

package scrape;


message GetScrapeRequest {
    repeated string queries = 1;
    string user_query = 2;
}
